dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkspaceItemRestRepositoryIT.java

```diff
@@ -119,11 +113,6 @@ public class WorkspaceItemRestRepositoryIT extends AbstractControllerIntegration
     private Group embargoedGroup2;
     private Group anonymousGroup;
 
-    public static final String REST_SERVER_URL = "http://localhost/api/";
```

Comment:
why are we deleting this test?

Reply:
Because it wasn't used anywhere, I do now know why it was there because class `AbstractControllerIntegrationTest` from where the `WorkspaceItemRestRepository` is extended has this property `REST_SERVER_URL` too. So this removal is safe.